### DISCUSSION TITLE
remove members-tab from dashboard

### DISCUSSION THREAD

**Discussion by man-tax** [illacloud/illa-builder#1118] [Ideas]
hey everyone,

can the members-tab be removed from the dashboard (after login) for viewers? reason is I would not want viewers to see a list of all the members in the team.

Thanks

**Comment by wenzhuo2022** [comment]
Hello, you can close “Allow editors and viewers to manage members” in the member panel. After closing, editors and viewers will have no access to the member list. If you want to share apps with users, not in your team, try to set the app public. In a few days, we will divide the editor and viewer settings into two switches. 
<img width="713" alt="image" src="https://user-images.githubusercontent.com/112603073/224596419-d7a143ba-5a7c-443a-af33-68e3e123d764.png">
